### PR TITLE
fixed screen sizes

### PR DESCRIPTION
**Pull request by igniti0n**

### CODE REVIEW

File: Weather App/UI/HomeVC/HomeView.swift [hunk 1]
```diff
@@ -31,17 +31,12 @@ class HomeView : UIView {
...
-            .font : UIFont.systemFont(ofSize: 30.0 - 4, weight: .bold),
-            .foregroundColor : UIColor.white
-        ]
-    private let upAttributes :  [NSAttributedString.Key : Any]  =
-        [
-            .font : UIFont.systemFont(ofSize: 30.0 - 8),
-            .foregroundColor : UIColor.white
-        ]
+    private let normalFontSize: CGFloat = {
+        UIScreen.main.bounds.height > 600 ? 30 : 24
```
Comment (IvanKulundzic) [inline Weather App/UI/HomeVC/HomeView.swift]: I'm not sure I understand the logic here for deciding on the font size. Usually, you get font size from design.

File: Weather App/UI/SearchVC/SearchViewController.swift [hunk 2]
```diff
@@ -65,13 +65,15 @@ extension SearchViewController {
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
          let cell = tableView.dequeueReusableCell(withIdentifier: "cell",
                                                   for: indexPath)
+         let cellFontSize: CGFloat = UIScreen.main.bounds.height > 600 ? 30 : 24
```
Comment (IvanKulundzic) [inline Weather App/UI/SearchVC/SearchViewController.swift]: Could you do the design/setup of the cell in a different place? Can the cell iteself have a setup method for its setup?

**Review by IvanKulundzic — CHANGES_REQUESTED** [review]

File: Weather App/UI/SearchVC/SearchViewController.swift (hunk 2 above)
Comment (igniti0n) [reply Weather App/UI/SearchVC/SearchViewController.swift]: Yes, I can move this designing/setup into a method of the cell class itself, I will move it.

File: Weather App/UI/HomeVC/HomeView.swift (hunk 1 above)
Comment (igniti0n) [reply Weather App/UI/HomeVC/HomeView.swift]: I changed it to fit better for smaller screen sizes, if font should not be changed I will look for different solutions, maybe changing the available space for the text, as the height goes smaller, so it resizes itself if the text can not fit?

Comment (igniti0n) [reply Weather App/UI/HomeVC/HomeView.swift]: I removed it, no need for that, also yes the designer will give the font size and that should be used.

File: Weather App/UI/HomeVC/HomeView.swift [hunk 3]
```diff
@@ -62,6 +57,12 @@ class HomeView : UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public func adjustFont(){
```
Comment (IvanKulundzic) [inline Weather App/UI/HomeVC/HomeView.swift]: Why is this needed?

File: Weather App/UI/HomeVC/HomeView.swift [hunk 4]
```diff
@@ -62,6 +57,12 @@ class HomeView : UIView {
...
+        print("hF: ", humidityLabel.font)
+        print("pF: ", pressureLabel.font)
+        print("wF: ", windLabel.font)
+    }
+    
     public func updateWeatherView(weather: Weather){
```
Comment (IvanKulundzic) [inline Weather App/UI/HomeVC/HomeView.swift]: Why is this method `public`?

**Review by IvanKulundzic — CHANGES_REQUESTED** [review]

File: Weather App/UI/HomeVC/HomeView.swift (hunk 3 above)
Comment (igniti0n) [reply Weather App/UI/HomeVC/HomeView.swift]: Not needed at all, it was supposed to be deleted I forgot to delete it, I deleted it now.

File: Weather App/UI/HomeVC/HomeView.swift (hunk 4 above)
Comment (igniti0n) [reply Weather App/UI/HomeVC/HomeView.swift]: I moved it and updating search method, into an extension. Methods need to be able to be accessed in ViewController to be called when view needs to be updated with new values. For ex:

`        homeViewModel.onSearchWeatherFetch = { [weak self] weather in
            DispatchQueue.main.async {
                self?.homeView.updateWeatherView(weather: weather)
            }
        }
`

Comment (IvanKulundzic) [reply Weather App/UI/HomeVC/HomeView.swift]: Is `public` keyword needed here? Would it work without it?

Comment (igniti0n) [reply Weather App/UI/HomeVC/HomeView.swift]: Yes, it is unnecessary and not needed, I removed it in new commit.

**Review by IvanKulundzic — APPROVED** [review]
